microprot/scripts/split_search.py

```diff
@@ -509,27 +520,27 @@ def mask_sequence(hhsuite_fp, fullsequence_fp, subsequences_fp=None,
 
     # read the original protein file, used to run HHsearch
     p = Protein.read(fullsequence_fp, seq_num=1)
-    queryname = p.metadata['id'] + ' ' + p.metadata['description']
+    queryname = [p.metadata['id'], p.metadata['description']]
```

Comment:
maybe we should have more speaking variable names for those two fields of the queryname?